packages/core/src/routes/applications/application.openapi.json

```diff
@@ -26,6 +26,27 @@
       "post": {
         "summary": "Create an application",
         "description": "Create a new application with the given data.",
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "properties": {
+                  "protectedAppMetadata": {
+                    "description": "The data for protected app.",
+                    "properties": {
+                      "subDomain": {
+                        "description": "The subdomain prefix."
+                      },
+                      "origin": {
+                        "description": "The origin of target website."
+                      }
+                    }
```

Comment:
nit: would it be better to provide some examples for these properties?

Reply:
updated